package.json

```diff
@@ -36,9 +36,9 @@
     "atom-linter": "^4.6.1",
     "atom-package-deps": "^4.0.1",
     "cosmiconfig": "^1.1.0",
-    "lazy-req": "^1.1.0",
-    "stylelint": "6.3.2",
-    "stylelint-config-standard": "^7.0.0"
+    "deep-assign": "^2.0.0",
```

Comment:
It looks like your package.json changes are simply a reversion to a much older version... `deep-assign` was moved away from since it has many bugs.